### DISCUSSION TITLE
How to use the BackupChecks with a S3 disk

### DISCUSSION THREAD

**Discussion by craigpotter** [spatie/laravel-health#214] [Q&A] [answered]
How would you use this with a S3 disk? 
Currently you can use the locatedAt() but that only accepts a glob. 

Am I right in thinking this check would only work for local backups?

**Comment by freekmurze** [comment, marked as answer]
This now only works for local backups indeed.

I welcome a PR that would add a `onDisk($diskName)` method. The check would then have to loop through all files on that disk (I don't think there is a `glob` like function for disks.
